### PR TITLE
Update README for 1.13 release

### PR DESCRIPTION
**Pull request by erpett**
Updating Readme for 1.13.0 will then port to release branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,16 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
...
 <img src="https://github.com/Microsoft/vscode-mssql/raw/main/images/mssql-demo.gif" alt="demo" style="width:480px;"/>
 
+## Version 1.13.0
+* Release date: February 24, 2022
+* Release status: GA
+
+## What's new in 1.13.0
+* Adds object explorer table integration for SQL bindings with Azure Functions
+* Updated SqlParser to fix some intellisense issues
+* Fix decimal datatype handling
```
Comment (Charles-Gagnon) [inline README.md:31]: For fixes it's generally nice to link the issue as well. For this one we don't have a reported issue in this repo so let's just link the ADS one 

Fixes [#275](https://github.com/microsoft/azuredatastudio/issues/275) - Error handling large decimal values

File: README.md [hunk 2]
```diff
@@ -21,6 +21,16 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
...
+* Fixed BatchParser execution error caused by missing resource files
```
Comment (Charles-Gagnon) [inline README.md:32]: Fixes [#17221](https://github.com/microsoft/vscode-mssql/issues/17221) - BatchParser execution error caused by missing resource files

File: README.md [hunk 3]
```diff
@@ -21,6 +21,16 @@ See [the SQL developer tutorial] to develop an app with C#, Java, Node.js, PHP,
 
 <img src="https://github.com/Microsoft/vscode-mssql/raw/main/images/mssql-demo.gif" alt="demo" style="width:480px;"/>
 
+## Version 1.13.0
+* Release date: February 24, 2022
+* Release status: GA
+
+## What's new in 1.13.0
+* Adds object explorer table integration for SQL bindings with Azure Functions
```
Comment (Charles-Gagnon) [inline README.md:29]: `Adds context menu item for object explorer table nodes to create an Azure Function with SQL binding for that table`

@lucyzhang929 @chlafreniere @dzsquared This look better to you?

Comment (erpett) [reply README.md:29]: I copied this from the text Drew had that Chris pointed me towards https://github.com/microsoft/vscode-mssql/issues/17263

**Review by Charles-Gagnon — APPROVED** [review]

**Pull request merged**